config/locales/en.yml

```diff
@@ -52,6 +52,17 @@ en:
         intro_title: Introduction title
         introduction: Introduction
         main_image: Introduction image
+        tab_body_description: The body of the newsletter can contain up to 4 events.
+          The purpose of the format is for events (all types of sessions) and/or news
+          (you can be imaginative and think of other ways to use it) in agenda format.
+          It is not necessary to fill in all 4 events, if you do less it also appears
+          in a good format.
```

Comment:
```suggestion
        tab_body_description: The body of the newsletter can contain up to 4 events. The format is best suited for events and/or news to conform an agenda of what's coming. Of course you can let your imagination drive you to think on other ways to use it. It is not necessary to fill in all 4 events, missing entries won't appear in the newsletter. We do recommend though to fill 2 or 4 entries instead of an odd number of events.
```